homeassistant/components/sensor/ebox.py

```diff
@@ -62,25 +62,23 @@
 })
 
 
-def setup_platform(hass, config, add_devices, discovery_info=None):
+@asyncio.coroutine
+def async_setup_platform(hass, config, async_add_devices, discovery_info=None):
     """Set up the EBox sensor."""
     username = config.get(CONF_USERNAME)
     password = config.get(CONF_PASSWORD)
 
-    try:
-        ebox_data = EBoxData(username, password)
-        ebox_data.update()
-    except requests.exceptions.HTTPError as error:
-        _LOGGER.error("Failed login: %s", error)
-        return False
+    httpsession = hass.helpers.aiohttp_client.async_get_clientsession()
+    ebox_data = EBoxData(username, password, httpsession)
 
     name = config.get(CONF_NAME)
 
     sensors = []
     for variable in config[CONF_MONITORED_VARIABLES]:
+        _LOGGER.error("EBOX %s", variable)
         sensors.append(EBoxSensor(ebox_data, variable, name))
 
-    add_devices(sensors, True)
+    async_add_devices(sensors, False)
```

Comment:
Why did you replace `True` with `False` here. Do you not want the sensors to automatically fill their value on startup?